system.go

```diff
@@ -106,3 +106,96 @@ func (s *System) CreateSnapshotConsistencyGroup(
 
 	return &snapResp, nil
 }
+
+// GetMDMClusterDetails returns MDM cluster details
+func (s *System) GetMDMClusterDetails() (*types.MdmCluster, error) {
```

Comment:
Hi @shenda1, does these functionality work with powerflex system v4.0 as well? Can you share some results where the details for 3,6 MDMs and 4.0 MDMs are retrieved.
Thanks.

Reply:
Hi @khareRajshree The changes are tested only w.r.t PowerFlex version 3.6 as per the Terraform support matrix.